src/openms_gui/source/VISUAL/DataSelectionTabs.cpp

```diff
@@ -61,24 +61,24 @@ namespace OpenMS
     : QTabWidget(parent),
     spectra_view_widget_(new SpectraViewWidget(this)),
     id_view_widget_(new SpectraIdentificationViewWidget(Param(), this)),
-    spectraview_behavior_(new TOPPViewSpectraViewBehavior(tv)),
-    idview_behaviour_(new TOPPViewIdentificationViewBehavior(tv, id_view_widget_)),
+    spectraview_behavior_(new TVSpectraViewController(tv)),
```

Comment:
maybe rename variable to 
`spectraview_controller_`
`idview_controller_`
for consistency?